### ISSUE TITLE
[CODE] Account for Other Clan temperament in event filtering

### ISSUE THREAD

**Issue by scribblecrumb** [ClanGenOfficial/clangen#2429]
**Description**
Other Clan temperaments don't currently have an affect on the kinds of events they feature in.  It would be nice to add the ability to specify allowed temperaments for the other clan (and possibly the player clan's temperament?) for both Patrols and ShortEvents.

**Comment by flamedash2358** [comment]
Going to do this, (may have forgotten the assign command atm)

**Comment by j-gynn** [comment]
Unassigning so that someone else can take this on as it's been nearly a year.

**Comment by AdaTuanaDonmez** [comment]
Hi! I am a student in the CodePath AI301 program and I would like to work on this issue. I will start by reading through the existing patrol and short event code this week to understand how other clans and temperaments are currently represented and used, then explore how I might add the ability to specify allowed temperaments for the other clan (and possibly the player clan's temperament) in both Patrols and ShortEvents. Any pointers on where the relevant code lives, how clan temperaments are currently modeled, or what a good implementation looks like in this project would be really helpful!

**Comment by archanyhm** [comment]
> Hi! I am a student in the CodePath AI301 program and I would like to work on this issue. I will start by reading through the existing patrol and short event code this week to understand how other clans and temperaments are currently represented and used, then explore how I might add the ability to specify allowed temperaments for the other clan (and possibly the player clan's temperament) in both Patrols and ShortEvents. Any pointers on where the relevant code lives, how clan temperaments are currently modeled, or what a good implementation looks like in this project would be really helpful!

No. We don't allow AI contributions. Please let the administration of your program know that we do not wish to participate.
